std_msgs/Header header
int32 num
float32[] x
float32[] y